### PR TITLE
Docs/modify sto data procedure

### PR DESCRIPTION
**Pull request by victorwiebe**

### CODE REVIEW

File: src/procedures/ModifyTieredStoData.ts [hunk 1]
```diff
@@ -33,9 +36,35 @@ export const createTieredStoFactoryRefreshResolver = (
   return undefined;
 };
 
+/**
+ * Procedure to modify data associated to a Tiered STO.
```
Comment (monitz87) [inline src/procedures/ModifyTieredStoData.ts]: Procedure that modifies the configuration parameters of a Tiered STO

File: src/procedures/ModifyTieredStoData.ts [hunk 2]
```diff
@@ -33,9 +36,35 @@ export const createTieredStoFactoryRefreshResolver = (
...
+ * Transactions include times, funding type, oracles, tiers, limits, and addresses
```
Comment (monitz87) [inline src/procedures/ModifyTieredStoData.ts]: Delete this line

File: src/procedures/ModifyTieredStoData.ts [hunk 3]
```diff
@@ -33,9 +36,35 @@ export const createTieredStoFactoryRefreshResolver = (
...
+ */
 export class ModifyTieredStoData extends Procedure<ModifyTieredStoDataProcedureArgs> {
   public type = ProcedureType.ModifyTieredStoData;
 
+  /**
```
Comment (monitz87) [inline src/procedures/ModifyTieredStoData.ts]: Delete the empty lines between the items

File: src/procedures/ModifyTieredStoData.ts [hunk 4]
```diff
@@ -33,9 +36,35 @@ export const createTieredStoFactoryRefreshResolver = (
...
+   * - Modify the start and end date of the STO
+   *
+   * - Modify the fundraise types of the STO, whether they be ETH, POLY, or Stablecoin
```
Comment (monitz87) [inline src/procedures/ModifyTieredStoData.ts]: `- Modify the fundraise types of the STO`

File: src/procedures/ModifyTieredStoData.ts [hunk 5]
```diff
@@ -33,9 +36,35 @@ export const createTieredStoFactoryRefreshResolver = (
...
+   *
+   * - Modify the oracles of the STO including the denominated currency, as well the POLY and ETH oracle addresses
```
Comment (monitz87) [inline src/procedures/ModifyTieredStoData.ts]: `- Modify the custom currency symbol and the oracles used to convert POLY and ETH to said currency`

File: src/procedures/ModifyTieredStoData.ts [hunk 6]
```diff
@@ -33,9 +36,35 @@ export const createTieredStoFactoryRefreshResolver = (
...
+   *
+   * - Modify the tier information of the STO including rates, tokens per tier, and discount information
```
Comment (monitz87) [inline src/procedures/ModifyTieredStoData.ts]: `- Modify the STO's tiers (rates, tokens per tier and discounts when buying with POLY)`

File: src/procedures/ModifyTieredStoData.ts [hunk 7]
```diff
@@ -33,9 +36,35 @@ export const createTieredStoFactoryRefreshResolver = (
...
+   *
+   * - Modify limits of the STO including minimum investment and non accredited invesment limits
```
Comment (monitz87) [inline src/procedures/ModifyTieredStoData.ts]: `- Modify investment limits (min investment, max invested for non-accredited investors)`

File: src/procedures/ModifyTieredStoData.ts [hunk 8]
```diff
@@ -33,9 +36,35 @@ export const createTieredStoFactoryRefreshResolver = (
...
+   *
+   * - Modify STO addresses including the treasuryWallet, wallet, and stable coin adddresses
```
Comment (monitz87) [inline src/procedures/ModifyTieredStoData.ts]: `- Modify treasury wallet, wallet for unsold tokens and stable coin addresses`

File: src/procedures/ModifyTieredStoData.ts [hunk 9]
```diff
@@ -33,9 +36,35 @@ export const createTieredStoFactoryRefreshResolver = (
...
+   *
+   * - After the last transaction takes place, refresh the tiered sto entity factory information on the SDK
```
Comment (monitz87) [inline src/procedures/ModifyTieredStoData.ts]: Delete this line

File: src/procedures/ModifyTieredStoData.ts (hunk 3 above)
Comment (monitz87) [reply src/procedures/ModifyTieredStoData.ts]: Also, after the items and before the fail notes, add 

`Only transactions that will effectively present changes will be submitted`

File: src/procedures/ModifyTieredStoData.ts [hunk 10]
```diff
@@ -33,9 +36,35 @@ export const createTieredStoFactoryRefreshResolver = (
...
+   *
+   * Note that this procedure will fail if:
+   * - The STO has not been enabled or has been archived
+   * - The STO has already started
+   * - A custom currency is used with a Tiered STO on version 3.0.0
```
Comment (monitz87) [inline src/procedures/ModifyTieredStoData.ts]: `- Attempting to use a custom currency on an STO with version is 3.0.0 or lower`

File: src/procedures/ModifyTieredStoData.ts [hunk 11]
```diff
@@ -33,9 +36,35 @@ export const createTieredStoFactoryRefreshResolver = (
...
+   * - There is nothing to modify, based on the STO state and the parameters given
```
Comment (monitz87) [inline src/procedures/ModifyTieredStoData.ts]: `- The supplied parameters don't represent any changes in the STO`

**Comment by victorwiebe** [comment]
Should be okay for another review

File: src/procedures/ModifyTieredStoData.ts [hunk 12]
```diff
@@ -37,32 +37,26 @@ export const createTieredStoFactoryRefreshResolver = (
...
-   *
-   * - After the last transaction takes place, refresh the tiered sto entity factory information on the SDK
+   * Only transactions that will effectively present changes will be submitted
    *
    * Note that this procedure will fail if:
    * - The STO has not been enabled or has been archived
    * - The STO has already started
-   * - A custom currency is used with a Tiered STO on version 3.0.0
-   * - There is nothing to modify, based on the STO state and the parameters given
+   * - Attempting to use a custom currency on an STO with version is 3.0.0 or lower
```
Comment (monitz87) [inline src/procedures/ModifyTieredStoData.ts]: Remove "is"

**Pull request merged**